apps/app/src/components/Vault/VaultButtons.tsx

```diff
@@ -1,34 +1,53 @@
 import { Vault } from '@generationsoftware/hyperstructure-client-js'
 import { useUserVaultShareBalance } from '@generationsoftware/hyperstructure-react-hooks'
-import { DepositButton, DeprecatedVaultTooltip, WithdrawButton } from '@shared/react-components'
+import {
+  DelegateButton,
+  DepositButton,
+  DeprecatedVaultTooltip,
+  WithdrawButton
+} from '@shared/react-components'
 import classNames from 'classnames'
 import { useTranslations } from 'next-intl'
+import Image from 'next/image'
 import { Address } from 'viem'
 import { useAccount } from 'wagmi'
 
 interface VaultButtonsProps {
   vault: Vault
+  isVaultPage?: boolean
```

Comment:
Instead of providing a location, it'd be more helpful and reusable to provide the change the devs would want to see. In this case I'd use something like "showDelegateButton", or if you want to display it by default, "hideDelegateButton", etc.

Reply:
Something that follows the "is" boolean standard might also be "isDelegateButtonShown", any of these work :)